src/goto-instrument/document_properties.cpp

```diff
@@ -125,14 +122,15 @@ std::string escape_html(const std::string &s)
 {
   std::string dest;
 
-  for(std::size_t i=0; i<s.size(); i++)
+  for(auto ch : s)
```

Comment:
Shouldn't all these generic string utilities defined in this file actually be moved?

Reply:
There is already a place for these in `util/string_utils.h`.